### DISCUSSION TITLE
zotero高级设置问题

### DISCUSSION THREAD

**Discussion by LHT-Liu** [zotero-chinese/wiki#115] [求助 / Q&A] [answered]
<img width="683" alt="image" src="https://github.com/zotero-chinese/wiki/assets/64388319/c24ee760-0d7a-4b88-ae37-0a330a8edf9d">
请问base directory什么功能，我好像一直没有涉及，我也没有使用zotfile等插件。利用坚果云同步的时候根什么有关系呢？

**Comment by northword** [comment, marked as answer]
https://www.zotero.org/support/preferences/advanced

可以参考一下官方文档的描述，这个设置适用于链接的附件，这个目录定义 Zotero 查找链接的附件时，起始查找的位置；

利用坚果云同步时，可以通过 WebDAV 同步，这种情况下，不需要更改这个设置。

一般情况下，`高级` 里的设置都不需要改，除非你明确知道这个设置的意义。

**Reply by LHT-Liu** [reply]
不太理解 ，哎😑
